atlp-rwanda/atlp-devpulse-fn
# Fix: Documents Preview

# Pull Request Description

## Feature: #247  Fix Documents preview

### What does this PR do?

This PR is for fixing how the preview of the documents uploaded by the applicants look like. Currently we had to  view them in the new tab opened after clicking on preview button, but for the purpose of Improving User Experience i added the modal that will open in the app. So the user clicks on the button to view and the modal opens then view the document and after click on the close button.

### How Can This Be Manually Tested?
Checkout into `fix-docs-preview`
1. Login as Trainee
- Click on `my application` then click on apply.
- Apply on the any Cycle available and upload all documents required.
- After applying get back to my application and click on track my application.
- You will view the documents list and click on any and view how the modal opens
- Click on close to close the modal
2. Login as an Admin
- Login as Admin and click on applications cycles
- Click on any specific cycle you want to review in
- For every applicant see in the attachement column so that you can view the document they uploaded. Click on select document and select any document and view it's details in the in app modal
- Click on close to close the modal

### Credentials:
**Trainee:**
email: jane@example.com
password:password123

**Admin:**
email: admin@example.com
password: password123

**Current**
<img width="1406" alt="Screenshot 2024-12-03 at 08 55 37" src="https://github.com/user-attachments/assets/acb6fdd9-69db-4e8f-a2ac-037fe70c765e">

**Updated**
<img width="1169" alt="Screenshot 2024-12-03 at 08 45 06" src="https://github.com/user-attachments/assets/c2185e41-7ae4-41d2-81dc-9d5db777708c">
